modules/@ergonode/core/src/models/mappers/gridDataMapper.js

```diff
@@ -156,23 +156,23 @@ export function getDefaultDataFromQueryParams($route) {
         query: {
             page = DEFAULT_PAGE,
             itemsPerPage = DATA_LIMIT,
-            filter = '',
-            advancedFilter = '',
-            field = '',
-            order = '',
+            filter,
+            advancedFilter,
+            field,
+            order,
```

Comment:
Default values were a better solution. 
Some properties have default values here and the rest is conditioned in the function arguments. 
two different approaches, mixed up.

Reply:
It cannot lend in defaults and being exported as whole object. Query might have part of those fields. I would stick to conditions cuz defaults arguments are not set when any value is null and they might be null.